sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -198,11 +213,19 @@ def sample(
         num_chains = self.num_chains if num_chains is None else num_chains
         init_strategy = self.init_strategy if init_strategy is None else init_strategy
         num_workers = self.num_workers if num_workers is None else num_workers
-        init_strategy_num_candidates = (
-            self.init_strategy_num_candidates
-            if init_strategy_num_candidates is None
-            else init_strategy_num_candidates
+        init_strategy_parameters = (
+            self.init_strategy_parameters
+            if init_strategy_parameters is None
+            else init_strategy_parameters
         )
+        if init_strategy_num_candidates is not None:
+            warn(
+                """Passing `init_strategy_num_candidates` is deprecated as of sbi
+                v0.19.0. Pass a dictionary containing init strategy kwargs instead."""
+            )
```

Comment:
Same as above:

Passing init_strategy_num_candidates is deprecated as of sbi v0.19.0. Instead, use `init_strategy_parameters={"num_candidate_samples": 1000}`